app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,7 +1,6 @@
 class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
   skip_before_action :verify_authenticity_token
   include OmniauthAuthenticationHelper
-  include InvitationsHelper
```

Comment:
I think... we had it so that when you are signing up to join a group whe can say so on the form.